src/app/components/engangsstonad/engangsstonad.less

```diff
@@ -58,27 +57,36 @@
 		margin-bottom: @marginLarge;
 	}
 
-  .datovelger__outer {
-    .skjemaelement__label {
-      margin-top: 12px;
-      font-weight: bold;
-    }
-  }
+	.datovelger__outer {
+		.skjemaelement__label {
+			margin-top: 12px;
+			font-weight: bold;
+		}
+	}
+
+	.radioPanelGruppe {
+		.skjema__legend {
+			font-size: 16px;
+			margin-bottom: 8px;
+		}
+	}
+
+	.radioPanelGruppe:not(:first-child) {
+		margin-top: 12px;
+	}
 
-  .radioPanelGruppe {
-    .skjema__legend {
-      font-size: 16px;
-      margin-bottom: 8px;
-    }
-  }
+	.radioPanelGruppe .radioPanel:not(:first-child) {
+		margin-top: 8px;
+	}
 
-  .radioPanelGruppe:not(:first-child) {
-    margin-top: 12px;
-  }
+	.stepWrapper {
+		display: flex;
+		justify-content: space-between;
+	}
 
-  .radioPanelGruppe .radioPanel:not(:first-child) {
-    margin-top: 8px;
-  }
+	.buffer {
+		width: 15%;
```

Comment:
Her er jeg ingen ekspert, men hva løser buffer her som vi ikke kan fikse med å bruke `width` eller `margin-right` på stegindikatoren?

Reply:
Slipper at marginer fungerer for en oppløsning, men ikke for en annen. Flex space-between vil aligne stegindikatoren korrekt siden den har ett 15% width element på hver sin side.